WordPress/twentytwentyfour
# Add H1 to templates

**Description**


Adds new H1 headings to templates that are missing them.
Update the heading level order where needed.

Partial for https://github.com/WordPress/twentytwentyfour/issues/54

**Testing Instructions**
View the update templates and check that the heading level is correct, that the H1 is present, and no heading levels are skipped.

**Note:**
If a template has a comments area, but the post or page does not have any comments, the heading level will skip from H1 directly to H3. The only way to work around this is to add a new H2 heading above the comments area.
This is because there is no way (that I know of) to change the level of the "Leave a reply" heading in the **Comments form** block. This extra heading is not part of the design, so I have left it as is for now, until it has been discussed...
